.ci.yaml

```diff
@@ -9,31 +9,31 @@ enabled_branches:
   - master
 
 targets:
-  - name: linux_analyze
+  - name: Linux analyze
```

Comment:
so are target names unused now?

Reply:
Yes (I forgot to cc you in https://github.com/flutter/cocoon/pull/1265). For posterity, the plan is to temporarily use builder name, once this has soaked, switch back to name, then deprecate the builder field.

Reply:
cool, SGTM